changelog.md

```diff
@@ -1,3 +1,76 @@
+<a name="5.1.0"></a>
+# 5.1.0 (2019-05-30)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v5.0.0...v5.1.0)
+
+## New contributors!
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Beng Eu <beng@bengeu.com>
+David Boyne <david.boyne@bglgroup.co.uk>
```

Comment:
this is a thing we should probably change about the release scripts. Even though it's in the git log, most people probably don't want their email in our release notes. In the past we would have done `@thebengeu and @boyney123` for these. We *could* include the user.name from the commits if folks want to, though.

(the release script might have to hit the github api for those? unless they're adding some commit metadata on squash in there somewhere)

Reply:
Hm yeah fair point. 

I always assumed the emails in my GitHub commits are there for spreading my exposure to recruiter spam! 😆